frontend/src/components/maps/leaflet/mapUtils.tsx

```diff
@@ -105,22 +155,32 @@ export const pointToLayer = (feature: ICluster, latlng: LatLngExpression): Layer
  * Get an icon type for the specified cluster property details (type, draft, erp, spp etc)
  */
 export const getMarkerIcon = (feature: ICluster) => {
-  const { propertyTypeId, projectStatus, classificationId } = feature?.properties;
-  if (
-    [
```

Comment:
this list of statuses is incomplete and is at constant risk of being out of sync with the db. We really want to know what workflow the property is in anyway not what status.